### ISSUE TITLE
Where to add specific Documentation (ros_gz_bridge): Humble - Garden

### ISSUE THREAD

**Issue by Uhrendoktor** [gazebosim/ros_gz#379]
I want to update the gz_ros_bridge README and add additional documentation for `ROS2 Humble` <-> `Gazebo Garden`. (replacing `ign` with `gz`, etc. )
This repository consists of a lot of branches, of whom some are not fully up to date with the development branch.

`caguero/humble_garden` sounds like the most fitting branch. the caveat is, that it's stale. `humble` would also be a possibility, but by default `gazebo fortress` is recommended for humble. The amount of branches also makes it difficult for newbies to find the correct one. I myself was struggling for quite some time.

In short: what is the intended / recommended way of adding specific documentation / examples information to this repository?

**Comment by azeey** [comment]
The Humble/Garden pairing is currently not recommended because it requires building `ros_gz` from source. We're currently [testing](https://github.com/gazebo-tooling/release-tools/issues/895) binary distribution, so we'll update the documentation once that is done. We've also been migrating documentation to https://gazebosim.org/docs/garden/ros_installation and we'll just point to that from the READMEs in `ros_gz`, so if you're still interested in contributing, you'll want to edit https://github.com/gazebosim/docs/blob/master/ros_installation.md
